tests/MSBuildDeviceIntegration/Tests/InstallTests.cs

```diff
@@ -301,20 +271,34 @@ public void ToggleFastDev ()
 
 				Assert.IsTrue (builder.Install (proj), "Second install should have succeeded.");
 
-				directorylist = string.Empty;
-				foreach (var dir in overrideDirs) {
-					var listing = RunAdbCommand ($"shell run-as {proj.PackageName} ls {dir}");
-					if (!listing.Contains ("No such file or directory"))
-						directorylist += listing;
-				}
-
+				directorylist = GetContentFromAllOverrideDirectories (proj.PackageName, true);
 				Assert.AreEqual ("", directorylist, "There should be no files in Fast Dev directories! Instead found: " + directorylist);
 
 				//Deploy one last time to verify install still works without the .__override__ directory existing
 				Assert.IsTrue (builder.Install (proj), "Third install should have succeeded.");
 			}
 		}
 
+		[Test]
+		public void LoggingPropsShouldCreateOverrideDirForRelease ()
+		{
+			AssertCommercialBuild ();
+			AssertHasDevices ();
+
+			var proj = new XamarinAndroidApplicationProject {
+				IsRelease = true,
+			};
+			proj.SetAndroidSupportedAbis ("armeabi-v7a", "x86");
+
+			using (var builder = CreateApkBuilder (Path.Combine ("temp", TestContext.CurrentContext.Test.Name))) {
```

Comment:
For new tests, I normally just put:
```suggestion
			using (var builder = CreateApkBuilder ()) {
```